src/main/java/edu/project1/ConsoleHangman.java

```diff
@@ -0,0 +1,104 @@
+package edu.project1;
+
+import java.util.Scanner;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+//import static java.lang.StringTemplate.STR;
+
+public class ConsoleHangman {
+
+    private final static Logger LOGGER = LogManager.getLogger();
+    final String constEXIT = "exit";
+    final String constFINISH = "finish";
+    final String constLOSE = "You lost!";
```

Comment:
`BUG` константы делаются так:
```suggestion
   static final String CONST_LOSE = "You lost!";
```